code/modules/shieldgen/shield_gen_external.dm

```diff
@@ -23,7 +23,7 @@
 		// Ignore station areas.
 		if (the_station_areas[T.loc])
 			continue
```

Comment:
Shuttle hangars may not always use plating as a base turf, in that case a shuttle would be covered in shields again. Can we do this by doing something like:
```suggestion
		if (the_station_areas[T.loc] || is_shuttle_area(T.loc))
			continue
```

Instead of / as well as what you've done to prevent shields spawning on platings.

Will need to define that helper.